src/Selenium2Library/keywords/_element.py

```diff
@@ -479,17 +479,27 @@ def simulate(self, locator, event):
     def press_key(self, locator, key):
         """Simulates user pressing key on element identified by `locator`.
 
-        `key` is either a single character, or a numerical ASCII code of the key
-        lead by '\\\\'.
+        `key` is either a single character, a numerical ASCII code of the key,\
+         or a NAMED KEY as described at https://selenium.googlecode.com/git/docs/api/py/webdriver/selenium.webdriver.common.keys.html
 
         Examples:
-        | Press Key | text_field   | q |
-        | Press Key | login_button | \\\\13 | # ASCII code for enter key |
-        """
-        if key.startswith('\\') and len(key) > 1:
-            key = self._map_ascii_key_code_to_key(int(key[1:]))
-        #if len(key) > 1:
-        #    raise ValueError("Key value '%s' is invalid.", key)
+        | Press Key | text_field   | q        |                                               |
+        | Press Key | login_button | \\\\13   | # ASCII code for Enter key (DEPRECATED)       |
+        | Press Key | login_button | 10       | # ASCII code for Return key                   |
+        | Press Key | nav_console  | ARROW_UP | # selenium.webdriver.common.keys ARROW_UP KEY |
+        
+        NAMED KEY value is new in Selenium2Library 1.7.3.
+        """
+        if len(key) > 1:
```

Comment:
The one thing we have to consider with this approach is that users will no longer be able to specify 0-9 ASCII codes. The only ones that I think would be remotely important in that range are 8 and 9 (backspace and tab, respectively). 

Is this an issue? They could simply use ```BACKSPACE``` or ```TAB``` to do the same thing, so I'm not necessarily of the opinion that it really matters.